src/components/ButtonGroup/GroupButton/index.test.js

```diff
@@ -1,15 +1,14 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { render, screen } from '@testing-library/react';
 
 import GroupButton from '.';
 
 const Icon = () => <span>🌭</span>;
 const Wrapper = (props) => <div {...props} data-test="snapshot-wrapper" />;
 
 describe('GroupButton component', () => {
-  it('matches snapshot (set all props)', () => {
-    expect(
-      shallow(<GroupButton value="Feed" Icon={Icon} isActive={true} Wrapper={Wrapper} />)
-    ).toMatchSnapshot();
+  it('renders component', () => {
+    render(<GroupButton value="Feed" Icon={Icon} isActive={true} Wrapper={Wrapper} />);
+    expect(screen.getByRole('button')).toHaveTextContent('🌭');
```

Comment:
It might be better to query on both the button and the name, and then the expectation itself can be simpler?

`expect(screen.getByRole('button', { name: '🌭' })).toBeInTheDocument()`

I'm not sure if this works but if it does, I think it's maybe a bit more clearer and straightforward.

Reply:
it does work, i didn't know this.

and you're right, it does look better, changed it to this.